doc/nrf/getting_started/installing.rst

```diff
@@ -16,11 +16,12 @@ For instructions on automatic installation, see the :ref:`gs_assistant` page.
 
 To manually install the |NCS|, you must first install all the required tools and clone the |NCS| repositories.
 
+The |NCS| toolchain is broader in scope than the Zephyr OS toolchain.
```

Comment:
The Zephyr "toolchain" is actually called "Zephyr SDK":
https://docs.zephyrproject.org/latest/develop/toolchains/zephyr_sdk.html
please use that name in order to avoid confusing users.

The  Nordic toolchain includes the Zephyr SDK and then adds on top of it everything else required to build an NCS app.

@martingalvan-nordic please expand if required

Reply:
@carlescufi , thanks for the comment. I've applied the edits. Please check again at your earliest convenience.